index.bs

```diff
@@ -1527,6 +1545,40 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Presentation Preference enumeration (enum <dfn enum>AttestationPresentationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPresentationPreference}} enum to specify their preference regarding attestation types used
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPresentationPreference {
+        "none",
+        "indirect",
+        "direct"
+    };
+</pre>
+
+<ul>
+    <li><dfn>none</dfn> - indicates that the [=[RP]=] is not interested in attestation, and prefers fast and reliable credential
+      creation over obtaining a verifiable attestation. The client MAY replace the attestation statement generated
+      by the authenticator with a meaningless client-generated attestation statement in this case, for example in order to prevent
```

Comment:
I suggest something to the effect of:

>The client MAY replace the attestation statement generated by the authenticator with any value - including `null`, an empty object or a meaningless client-generated attestation statement - in this case, for example in order to...

I don't think we need to prescribe that the client has to generate a nonsense attestation statement when no attestation statement at all has equivalent security properties.